src/extension.ts

```diff
@@ -110,26 +112,43 @@ function initCommand(context: vscode.ExtensionContext, commandId: string, callba
 function initAsyncCommand(context: vscode.ExtensionContext, commandId: string, callback: (...args: any[]) => Promise<any>) {
 	context.subscriptions.push(vscode.commands.registerCommand(commandId, async (...args: any[]) => {
 		const start = Date.now();
-		let result = 'Succeeded';
-		let errorData: string = '';
+		let properties: { [key: string]: string; } = {};
+		properties.result = 'Succeeded';
+		let errorData: ErrorData | undefined = null;
+		const output = util.getOutputChannel();
+		context.subscriptions.push(output);
 
 		try {
 			await callback(...args);
 		} catch (err) {
-			result = 'Failed';
-			errorData = util.errToString(err);
+			properties.result = 'Failed';
 			if (err instanceof UserCancelledError) {
-				result = 'Canceled';
+				properties.result = 'Canceled';
 			}
 			else if (err instanceof Error) {
-				vscode.window.showErrorMessage(err.message);
+				errorData = new ErrorData(err);
+				output.appendLine(errorData.errorType);
```

Comment:
No need to output the errorType?

Reply:
The DocDBErrorTypes seemed useful one word summaries, for eg., 'Conflict' . I can remove them if they are stylistically inconsistent.

Reply:
It's not _always_ useful, though. For example, sometimes errorType is just 'string'

I have a slight preference not to print it. However, if you disagree I would at least do this instead:
```
output.appendLine(`Error type: ${errorData.errorType}`);
```